css/modules.css

```diff
@@ -52,17 +64,109 @@ Later, this will be put into @media for larger screens
 =============================================*/
 .slideshow-div{
   margin: 0px 5%;
+  width: 100%;
   overflow: hidden;
-  width: 1214.99px;
 }
 #slideshow-article .slideshow{
-  display: block;
-  width: 7290px;
+  /*display: block;*/
+  width: 90%;
   margin: 10px 0px;
   padding: 0px;
 }
 #slideshow-article .my-slides{
+  display: inline;
   float: left;
   list-style-type: none;
-  max-width: 100%;
+  /*max-width: 100%;
+  min-height: 150px;
+  max-height: 300px;*/
+}
+
+/*Desktop Compatibility
+=========================================================================================*/
+@media screen and (min-width: 740px) {
+  /*Adds an rgba background with opacity to every div element on the page
+  =============================================*/
+  .divs {
+    border: solid black;
+    margin: 30px 0px;
+    background-color: rgba(200, 200, 200, 0.70);
+  }
+  /*header and nav-bar special borders, fixed nav bar
+  =============================================*/
+  .nav-bar {
+    display: block;
+    top: 0px;
+    position: fixed;
+    min-width: 85%;
+    height: 5%;
+    border: 2px dashed blue;
+    margin-top: 0px;
+    margin-bottom: 1px;
+    background-color: rgba(200, 200, 200, 0.70);
+  }
+  body .header {
+    display: block;
+    position: absolute;
+    min-width: 85%;
+    height: 7.5%;
+    margin-bottom: 100px;
+    padding: 0;
+    text-align: left;
+    border: 2px dashed #FF0000;
+    background-color: rgba(200, 200, 200, 0.70);
+  }
+  /*table of contents for the projects floated to the left of the slideshow
+  =============================================*/
+  .table-of-contents{
+    /*float: left;*/
+    display: block;
+    text-align: center;
+    max-width: 22.5%;
+    font-size: 35px;
+    margin-top: 15vh;
+  }
+  .table-of-contents li{
+    margin-left: 40px;
+    text-align: left;
+    list-style: lower-roman;
+    font-size: 25px;
+  }
+  #slideshow-article {
```

Comment:
When a user starts to shrink your page from full screen, you images go from the side of the aside to under it. You should find a way to keep your aside and images side by side,  if that is what you want, until your aside stretches across the top. Let me know if you have any questions about this.